package.json

```diff
@@ -76,7 +80,7 @@
     "sinon": "^1.16.1",
     "tslint": "^3.11.0",
     "tslint-react": "^0.4.0",
-    "typescript": "^1.9.0-dev.20160617-1.0",
+    "typescript": "^2.1.0-dev.20160716",
```

Comment:
Sweet - new version.  That has `async/await` I assume.  Will update my other references to this too.